windows.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2017 The CRT Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:generate go run generator.go
+
+// +build windows
+
+package crt
+
+import (
+	"fmt"
+	"os"
+	"sync/atomic"
+	"syscall"
+	"unsafe"
+
+	"github.com/cznic/memory"
+)
+
+// TODO: implement a generic wide string variant of this
+// GoUTF16String converts a wide string to a GOString using
+// windows-specific implementations in go's syscall package
+func GoUTF16String(s uintptr) string {
+	ptr := (*[1 << 20]uint16)(unsafe.Pointer(s))
```

Comment:
Please introduce a `rawwords`  or alike type, similar to the architecture specific `rawmem` type instead of `[1<<20]`.

Reply:
@steffengy This one seems to got missed in the second round?

Reply:
Not critical anyway.

Reply:
Yeah, I queued it up behind the sqlite2go changes, since I want to look at them first, and I'll probably have to think about that a bit (If it's even correct like that).